geotrek/common/parsers.py

```diff
@@ -254,6 +275,8 @@ def parse_field(self, row, dst, src, updated, non_field):
             val = self.get_val(row, dst, src)
         if non_field:
             modified = self.parse_non_field(dst, src, val)
+        elif self.fill_empty_translated_fields and dst in self.translated_fields:
+            modified = self.parse_translated_field(dst, src, val)
```

Comment:
By shunting out `self.parse_real_field` in the case of a translated field the `self.apply_filter` will not be call if a filter exists for that field.
I don't know if translated fields often have filters but not calling it breaks the parser filter hook interface.
I don't see a better way than just call it in `parse_translated_field`, maybe by wrapping the check for a filter method and the call in a new method. It would be better to add a test case with a filtered translated fields too.

Reply:
Exact, thanks for having seen this problem.

Reply:
Normally fixed